check-ostree.yaml

```diff
@@ -512,8 +512,6 @@
       block:
         - assert:
             that:
-              # Work around for issue https://github.com/virt-s1/rhel-edge/issues/1402
-              - result_dmesg_error.stdout_lines | length <= 4
```

Comment:
@yih-redhat Could you please keep this checking and skip your error log in command `dmesg` with `-v "xxxx"`? Thanks.

Reply:
@henrywang  Do you mean to change to this? The change will apply to all test scripts, do you want me to skip error log only for vmdk image?

[admin@vm-198-43 ~]$ dmesg --notime | grep -i "error\|fail" | grep -v "skipped" | grep -v "failover" | grep -v "ignition" | grep -v "failed to assign" || true
systemd[1]: sys-module-fuse.device: Failed to enqueue SYSTEMD_WANTS= job, ignoring: Unit sys-fs-fuse-connections.mount not found.
systemd[703]: /usr/lib/systemd/system-generators/podman-system-generator failed with exit status 1.


Comparing to error log before,
[admin@vm-198-164 ~]$ dmesg --notime | grep -i "error\|fail" | grep -v "skipped" | grep -v "failover" | grep -v "ignition" || true
pci 0000:00:16.4: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.5: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.6: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.7: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.0: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.1: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.2: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.3: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.4: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.5: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.6: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.7: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.0: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.1: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.2: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.3: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.4: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.5: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.6: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:18.7: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.5: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.4: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.3: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.2: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.1: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:17.0: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.7: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.6: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.5: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.4: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.3: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.2: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:16.1: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:15.7: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:15.6: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:15.5: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:15.4: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:15.3: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:15.2: BAR 13: failed to assign [io  size 0x1000]
pci 0000:00:15.1: BAR 13: failed to assign [io  size 0x1000]
systemd[1]: sys-module-fuse.device: Failed to enqueue SYSTEMD_WANTS= job, ignoring: Unit sys-fs-fuse-connections.mount not found.

Reply:
I added the checkpoint back and it will ignore multiple lines of errors like "pci 0000:00:15.1: BAR 13: failed to assign [io size 0x1000]", now it applies to all test script, please let me know if this is only for vmdk image.

dmesg --notime | grep -i "error\|fail" | grep -v "skipped" | grep -v "failover" | grep -v "ignition" | grep -v "failed to assign" || true

Reply:
Does `grep -v "BAR 13: failed to assign"` work?

Reply:
Yes, it works.

I changed the command to:
dmesg --notime | grep -i "error\|fail" | grep -v "skipped" | grep -v "failover" | grep -v "ignition" | grep -v "BAR 13":" failed to assign" || true 

Please be noted, the : after 'BAR 13' must be quoted, otherwise the ansible playbook will treat the content after : as command to run, which is not correct, the command should start with dmesg.